atef/widgets/config/utils.py

```diff
@@ -789,9 +789,11 @@ def get_relevant_pvs(
         for device_name in config.devices:
             dev = util.get_happi_device_by_name(device_name)
             for curr_attr in attrs:
-                pv = getattr(getattr(dev, curr_attr), 'pvname', None)
-                if pv:
+                try:
+                    pv = getattr(getattr(dev, curr_attr), 'pvname', None)
                     pv_list.append((pv, device_name + '.' + curr_attr))
```

Comment:
`getattr(..., "pvname", None)` can return None, and `pv_list` will include a `None` as the first item. Is that OK here?

Also, the `AttributeError` is probably only trying to catch `getattr(dev, cur_attr)`, so the try block should only include that code

Reply:
This should not return anything if pvname is none, good catch